tests/unit/test_data_interface.py

```diff
@@ -267,3 +282,59 @@ def testSearchReferences(self):
                 for repoReference, reference in utils.zipLists(
                         repoReferences, references):
                     self._assertEqual(repoReference, reference)
+
+    def testSearchRnaQuantificationSets(self):
+        self._testSearchMethodInContainer(
+            'getRnaQuantificationSets',
+            self._client.search_rna_quantification_sets,
+            self._repo.getDatasets())
+
+    def testSearchRnaQuantifications(self):
+        # TODO Can't use because of weird search_rna_quantifications signature
+        """
+        self._testSearchMethodInContainer(
+            'getRnaQuantifications',
+            self._client.search_rna_quantifications,
+            self._repo.allRnaQuantificationSets())
+        """
+        for rnaQuantificationSet in self._repo.allRnaQuantificationSets():
+            rnaQuantificationSetId = rnaQuantificationSet.getId()
+            repoRnaQuantifications = \
+                rnaQuantificationSet.getRnaQuantifications()
+            numRepoRnaQuantifications = len(repoRnaQuantifications)
+            pageSizes = self._getPageSizes(numRepoRnaQuantifications)
+            for pageSize in pageSizes:
+                self._client.set_page_size(pageSize)
+                datasetId = rnaQuantificationSet.getCompoundId().dataset_id
+                clientRnaQuantifications = list(
+                    self._client.search_rna_quantifications(
+                        datasetId, rnaQuantificationSetId))
+                for repoRnaQuantification, clientRnaQuantification in \
+                        utils.zipLists(
+                        repoRnaQuantifications, clientRnaQuantifications):
+                    self._assertEqual(
+                        repoRnaQuantification, clientRnaQuantification)
+
+    def testSearchExpressionLevels(self):
+        # TODO Can't use this because of weird getExpressionLevels signature
```

Comment:
@saupchurch Why does `SqliteRnaQuantification.getExpressionLevels` take a rnaQuantificationId as a parameter?  Couldn't it just use the class instance's id?